extensions/odoo_theme/static/scss/_font_awesome.scss

```diff
@@ -0,0 +1,947 @@
+/*!
+ *  Based on Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
+ *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
+ */
+/* FONT PATH
+ * -------------------------- */
+[class^="fa-"] {
```

Comment:
Why not use the library as is, and therefore the `.fa` class?

Is it to simplify the use of `::icon::` and avoid having different "tags" when you import other libraries ? (e.g. avoid having something like `:icon_fa:`, `:icon_oi:`)

If so, I agree, but with this selector, it will only work if you define the icon first:

```xml
:icon:`fa-code-fork text-success` <!-- will work ✅  -->
:icon:`text-success fa-code-fork` <!-- won't work ❌ -->
```

To fix that, you should use :
```suggestion
[class*="fa-"] {
```

Reply:
Yes the intention is to simplify both the `:icon:` role and the classes added to it to render an icon. For Font Awesome specifically, I wanted to allow the use of just the icon class name without needed to prefix the `fa` class and the icon class (e.g. `fa-code-fork` instead of `fa fa-code-fork`).

Great suggestion to update the selector 👍